### PR TITLE
docker bootstrap is removed

### PR DESCRIPTION
**Pull request by batikanu**
Docker bootstrap related parts are removed from the project

### CODE REVIEW

File: OS/CentOS/setup.sh [hunk 1]
```diff
@@ -12,30 +12,10 @@ gpgcheck=1
...
 sudo yum install -y docker-engine
 
 sudo systemctl enable docker
 sudo systemctl restart docker
 
-sudo systemctl enable docker-bootstrap
-sudo systemctl restart docker-bootstrap
-
 # install bridge-utils if not already installed
 rpm -qa | grep -qw bridge-utils || sudo yum -y install bridge-utils
```
Comment (cheld) [inline OS/CentOS/setup.sh]: I assume these can be removed, too?

**Comment by cheld** [comment]
One small comment. Other than that, LGTM

**Comment by batikanu** [comment]
PTAL

**Comment by cheld** [comment]
LGTM

**Pull request merged**